nomos-services/data-availability/dispersal/src/lib.rs

```diff
@@ -88,7 +88,7 @@ where
     type Settings = DispersalServiceSettings<Backend::Settings>;
     type State = NoState<Self::Settings>;
     type StateOperator = NoOperator<Self::State>;
-    type Message = DaDispersalMsg<Metadata>;
+    type Message = DaDispersalMsg<Metadata, Backend>;
```

Comment:
In this `impl` block is where the constrains should be.

Reply:
i will try this approach